evap/results/views.py

```diff
@@ -376,38 +401,58 @@ def evaluation_detail_parse_get_parameters(request, evaluation):
     if not evaluation.can_results_page_be_seen_by(request.user):
         raise PermissionDenied
 
-    view = request.GET.get("view", "public" if request.user.is_reviewer else "full")
-    if view not in ["public", "full", "export"]:
-        view = "public"
+    view_general_text = request.GET.get(
+        "view_general_text",
+        "full" if request.user.is_reviewer or request.user.is_responsible_or_contributor_or_delegate else "ratings",
```

Comment:
`request.user.is_responsible_or_contributor_or_delegate` is a general attribute of the account and not related to the current evaluation (you would need `evaluation.is_user_responsible_or_contributor_or_delegate(request.user)` for that). Could we just use `"full"` without conditions here because users who are not responsible, contributor, or delegate won't see the text answers and the choice buttons anyway?

@richardebeling asking for a second pair of eyes here

Reply:
Yes, sounds reasonable to me, and shouldn't have security implications.